docs/howto/operate/new-cluster/new-cluster.md

```diff
@@ -0,0 +1,204 @@
+(new-cluster:new-cluster)=
+# Add a new Kubernetes cluster
+
+This guide will walk through the process of adding a new cluster to our [terraform configuration](https://github.com/2i2c-org/infrastructure/tree/HEAD/terraform).
```

Comment:
Could we cross-link to a section in the docs that explains what terraform is and links to docs? (I think this page already exists somewhere, if it doesn't then creating that page doesn't need to be part of this PR)